site/content/en/docs/administration/advanced/update_guide.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Update guide'
+linkTitle: 'Update guide'
+weight: 11
+description: 'Instructions for updating CVAT deployed with docker compose'
+---
+
+<!--lint disable heading-style-->
+
+## About CVAT data volumes
+
+Docker volumes are used to store all CVAT data:
+
+- It is highly recommended backup all CVAT data before updating, follow the
+  [backup guide](/docs/administration/advanced/backup_guide/) and backup all CVAT volumes.
+
+- Update CVAT source code by any preferable way: clone with git or download zip file from GitHub.
+  Note that you need to download the entire source code, not just the `docker-compose` configuration file.
+  - With git:
+    Run the following commands inside CVAT folder
+    ```shell
+    git checkout develop
+    git pull
+    ```
+    Optionally checkout to specific version:
+    ```shell
+    git checkout v2.1.0
+    ```
+  - With wget:
+    ```shell
+    wget https://github.com/opencv/cvat/archive/refs/heads/develop.zip
+    unzip develop.zip -d cvat && mv cvat/cvat-develop/* cvat && rm -r cvat/cvat-develop
+    ```
+  Check the [installation guide](/docs/administration/basics/installation/) for details.
+
+- Update settings:
+  If `docker-compose.override.yml` configuration file is present in your installation,
```

Comment:
Again, I would redirect the user to the installation guide (a specific section). Here we need just basic instructions, but notify users about possible problems.